optuna/structs.py

```diff
@@ -18,6 +18,18 @@
 
 _logger = logging.get_logger(__name__)
 
+message = (
+    "The use of `structs` module is deprecated. "
+    "Please use the following way for each class. \n"
+    "`structs.StudyDirection` -> `study.StudyDirection`, \n"
+    "`structs.StudySummary`   -> `study.StudySummary`, \n"
+    "`structs.FrozenTrial`    -> `trial.FrozenTrial`, \n"
+    "`structs.TrialState`     -> `trial.TrialState`, \n"
+    "`structs.TrialPruned`    -> `exceptions.TrialPruned`."
```

Comment:
A comment on the English phrasing. How about the following?
> `structs` is deprecated. Classes have moved to the following modules. `structs.StudyDirection` -> ...

Reply:
Please correct me if I'm wrong but I don't see line breaks very often in warnings/error messages.

Reply:
I'm not familiar with the rule of line breaks in warning/error messages, but I think the warning looks squalid without line breaks. However, I will follow the convention.

Reply:
Did you mean to leave it as is? I think it's better to omit the line breaks as it would make string comparisons/parsing messages more complicated.

Reply:
Thank you for your clarification. Ok, I got it. I omit the line breaks.